### ISSUE TITLE
doesn't work, detectable by defender

### ISSUE THREAD

**Issue by xxopenxx** [0xb11a1/yetAnotherObfuscator#6]
as title said

**Comment by 0xb11a1** [comment]
Can you share the name of the executable that was detected so i can look at it

**Comment by prisonerr627** [comment]
> Can you share the name of the executable that was detected so i can look at it

SharpChrome also gets caught https://github.com/GhostPack/SharpDPAPI/tree/master/SharpChrome

**Comment by Darkmatterv2** [comment]
Yeah and so does njrat, neptune rat. All detected. even discord detects it as a virus.
